pkg/errors/errors.go

```diff
@@ -41,8 +41,21 @@ var ErrInvalidManifestVersion = RemediationError{
 	Remediation: "Delete `[manifest_version]` from the fastly.toml if present",
 }
 
+// ErrIncompatibleManifestVersion means the manifest_version defined is no
+// longer compatible with the current CLI version.
+var ErrIncompatibleManifestVersion = RemediationError{
+	Inner:       fmt.Errorf("the fastly.toml contains an incompatible manifest_version number"),
+	Remediation: "Update the `manifest_version` in the fastly.toml and refer to https://developer.fastly.com/reference/fastly-toml/#setup-information",
```

Comment:
@integralist I need to update this message so it points to the next CLI changelog that this code will be released under.